### DISCUSSION TITLE
Excluding vcs files from backup

### DISCUSSION THREAD

**Discussion by stefanopallicca-imagicle** [WeblateOrg/weblate#6425] [Q&A] [answered]
Hi everyone!
We bought an annual plan for backing up our weblate installation with weblate provisioned backup service.

Unfortunately, I find no way to configure our weblate installation in order to exclude the vcs/ directory, and now we have some useless 4+ GB of wasted space on weblate cloud.
I would only be interested in backing-up weblate data (i.e. database, users, settings, all the applicative stuff, to be clear).
No vcs backup is needed, as we have our codebase repository elsewhere.

Any way to achieve this?

**Comment by nijel** [comment, marked as answer]
The scope of backups is currently not configurable right now.
